db/schema.rb

```diff
@@ -39,40 +35,32 @@
     t.index ["phone_number"], name: "index_users_on_phone_number", unique: true
   end
 
-  create_table "voters", primary_key: "sos_id", id: :serial, force: :cascade do |t|
+  create_table "voters", primary_key: "reach_id", id: :string, force: :cascade do |t|
+    t.string "sos_id"
     t.string "last_name"
     t.string "first_name"
     t.string "middle_name"
+    t.string "suffix"
+    t.string "prefix"
     t.integer "age"
     t.string "gender"
+    t.string "email"
+    t.string "phone_country_code"
     t.string "primary_phone_number"
     t.string "voting_street_address"
+    t.string "voting_state"
     t.string "voting_city"
     t.string "voting_zip"
-    t.string "support_score"
-    t.string "vote_plan"
+    t.string "voter_registration"
+    t.json "voting_history"
     t.string "voting_status"
-    t.boolean "voted_general"
-    t.float "gotv_score"
+    t.integer "voting_address_id"
+    t.integer "voter_data_status"
+    t.integer "last_call_status", default: 0, null: false
+    t.integer "support_score"
     t.integer "household_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.integer "last_call_status", default: 0, null: false
-    t.integer "tier", default: 3, null: false
-    t.integer "tier_raw"
-    t.boolean "voted"
-    t.string "vote_location_name"
```

Comment:
So this could be useful later right? But I guess better practice would be to store polling locations in a separate table entirely, and link it based on zip or something. So I'm cool with getting rid of these fields